docs/user-interface/controls/blazorwebview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "BlazorWebView"
+description: "The .NET Multi-platform App UI (.NET MAUI) BlazorWebView control enables you to host a Blazor web app in your .NET MAUI app, and integrate the app with device features."
+ms.date: 08/19/2021
+---
+
+# BlazorWebView
+
+The .NET Multi-platform App UI (.NET MAUI) `BlazorWebView` is a control that enables you to host a Blazor web app in your .NET MAUI app. These apps, known as .NET MAUI Blazor apps, enable a Blazor web app to be integrated with native platform features and UI controls. The `BlazorWebView` control can be added to any page of a .NET MAUI app, and pointed to the root of the Blazor app. The Blazor components run natively in the .NET process and render web UI to an embedded web view control. .NET MAUI Blazor apps can run on all the platforms supported by .NET MAUI.
+
+> [!IMPORTANT]
+> Using the `BlazorWebView` control requires you to have installed the ASP.NET and web development workload in Visual Studio. In addition, to use the `BlazorWebView` control on Windows requires you to install the [WebView2](https://developer.microsoft.com/microsoft-edge/webview2/) runtime.
+
+`BlazorWebView` defines the following properties:
+
+- `HostPage`, of type `string?`, which defines the root page of the Blazor web app.
+- `RootComponents`, of type `ObservableCollection<RootComponent>`, which specifies the collection of root components that can be added to the control.
+
+The `RootComponent` class defines the following properties:
+
+- `Selector`, of type `string?`, which defines the CSS selector string that specifies where in the document the component should be placed.
+- `ComponentType`, of type `Type?`, which defines the type of the root component.
+- `Parameters`, of type `IDictionary<string, object?>?`, which represents an optional dictionary of parameters to pass to the root component.
+
+For information about Blazor apps, see [Introduction to ASP.NET Core Blazor](/aspnet/core/blazor/).
+
+> [!NOTE]
+> Existing Blazor components can be used in a .NET MAUI Blazor app by moving the code into the app, or by referencing an existing class library or package that contains the component.
+
+## Create a .NET MAUI Blazor app
+
+A .NET MAUI Blazor app can be created in Visual Studio by the **.NET MAUI Blazor app** template:
+
+:::image type="content" source="blazorwebview-images/blazorwebapp-template.png" alt-text=".NET MAUI Blazor app project template screenshot.":::
+
+This project template creates a multi-targeted .NET MAUI Blazor app that can be deployed to Android, iOS, Mac Catalyst, and Windows. The `BlazorWebControl` is defined in *MainPage.xaml* in the created project, and points to the root of the Blazor app:
```

Comment:
```suggestion
This project template creates a multi-targeted .NET MAUI Blazor app that can be deployed to Android, iOS, Mac Catalyst, and Windows. The `BlazorWebView` is defined in *MainPage.xaml* in the created project, and points to the root of the Blazor app:
```

Reply:
Another comment: Here it says `Mac Catalyst` but that's more of the app architecture, not the platform per se. The _platform_ is macOS. This is often discussed throughout MAUI in terms of where should it say Mac Catalyst / WinUI and where should it say macOS / Windows.